samcli/commands/build/build_context.py

```diff
@@ -650,34 +642,6 @@ def _collect_single_buildable_layer(
 
         resource_collector.add_layer(layer)
 
-    @staticmethod
```

Comment:
This functionality is moved to `Function` class and `FunctionBuildInfo` enum below.